pkgs/by-name/fa/fahrplan/package.nix

```
@@ -0,0 +1,28 @@
+{ lib, python3, fetchFromGitHub }:
+let
+  pname = "fahrplan";
+  owner = "dbrgn";
+  version = "1.1.2";
+in python3.pkgs.buildPythonApplication {
+  inherit pname version;
+  format = "setuptools";
```

```suggestion
  pyproject = true;
```
and specify `build-system`